source/build/GDK/MicrosoftGame.Config

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Game configVersion="0">
+  <Identity Name="41336MicrosoftATG.XboxLiveE2E" Publisher="CN=A4954634-DF4B-47C7-AB70-D3215D246AF1" Version="1.7.0.0" />

```

Comment:
>41336MicrosoftATG.XboxLiveE2E [](http://example.com/codeflow?start=18&length=29)

Taken from XSAPI APIRunner.  @JasonSa how do we create a new identity for the PlayFab test app?

Reply:
signin to partner center and make one.  you should have access to the account api runner used.  if not, ping me

Reply:
I do have access.  I created a new config here: [https://partner.microsoft.com/en-us/dashboard/products/9NF426QNMLZ5/overview](https://partner.microsoft.com/en-us/dashboard/products/9NF426QNMLZ5/overview).  It can't be used yet - still needs approval to enabel XboxLive and to publish to XDKS.1 sandbox.  Going to continue using XSAPI config for now and we can transition to this later if needed.